lua/entities/ent_jack_landmine/init.lua

```diff
@@ -142,11 +142,13 @@ function ENT:Use( activator )
     self:SetAngles( traceResult.HitNormal:Angle() )
     self:SetPos( traceResult.HitPos + traceResult.HitNormal )
 
-    if traceResult.Entity == game.GetWorld() then
```

Comment:
why was the world check removed

Reply:
used to freeze if the ent was the world, else parent to it
now it parents if it can, else freezes